app/player/player.view.html

```diff
@@ -45,21 +38,21 @@
 			    	<span class="glyphicon glyphicon-plus"></span>
 			    </button>
 			</span>
-		</div> <!-- input-group -->
+		</div> <!-- input-group --
 		<div role="alert" class="errorLength">
       <span class="error" ng-show="playerForm.namePlayer.$error.minlength">Too short!</span>
     </div>
-	</div> <!-- col-lg-6 -->
+	</div> <!-- col-lg-6 --
 	<div>
 		<ul class="list-group playerList">
 			<li class="list-group-item" ng-repeat="player in vm.players">
 				{{ player.name }}
 			</li>
 		</ul>
 	</div>
-
+-->
 	<div class="dartsButton">
-		<a class="btn btn-warning btn-lg" role="button" ng-click="vm.removeAllPlayers()" ng-show="vm.players.length >= 1">Remove Players</a>
-    <a class="btn btn-info btn-lg" role="button" href="#!/round" ng-show="vm.players.length >= 2">Start</a>
+		
+    <a class="btn btn-info btn-lg" role="button" href="#!/round" ng-show="vm.selectedPlayers.length >= 2" ng-click="vm.start()">Start</a>
```

Comment:
I know that is cool having a `2` directly in the view but what would it happen if the same value should be used in another place??